Dockerfile

```
@@ -0,0 +1,29 @@
+
+COPY . .
+
+WORKDIR /usr/src/starknet-rpc-tests/openrpc-testgen-runner
+
+ENV CARGO_TARGET_DIR=/usr/src/starknet-rpc-tests/openrpc-testgen-runner/target
+
+RUN cargo build --release --bin openrpc-testgen-runner
+
+FROM ubuntu:22.04
+
+RUN apt-get update && apt-get install -y \
```

Hadolint

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)